debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -149,11 +162,45 @@ public void configure(Configuration config, HistoryRecordComparator comparator)
     @Override
     public synchronized void start() {
         super.start();
+        createDatabaseHistoryTopic();
         if (this.producer == null) {
             this.producer = new KafkaProducer<>(this.producerConfig.asProperties());
         }
     }
 
+    private void createDatabaseHistoryTopic() {
+        final AdminClient admin = AdminClient.create(this.producerConfig.asProperties());
+        try {
+            short replicationFactor;
+
+            Optional<String> findTopic = admin.listTopics().names().get(KAFKA_QUERY_TIMEOUT.toMillis(), TimeUnit.MILLISECONDS).stream().filter(x -> topicName.equals(x)).findFirst();
+
+            if (!findTopic.isPresent()) {
+                // Find default replication factor
+                final Collection<Node> nodes = admin.describeCluster().nodes().get(KAFKA_QUERY_TIMEOUT.toMillis(), TimeUnit.MILLISECONDS);
+                if (nodes.isEmpty()) {
+                    throw new ConnectException("No brokers available to obtain default settings");
+                }
+                final Map<ConfigResource, Config> configs = admin.describeConfigs(Collections.singleton(new ConfigResource(ConfigResource.Type.BROKER, nodes.iterator().next().idString()))).all().get(KAFKA_QUERY_TIMEOUT.toMillis(), TimeUnit.MILLISECONDS);
+                if (configs.isEmpty()) {
+                    throw new ConnectException("No configs have been received");
+                }
+                final Config config = configs.values().iterator().next();
+                replicationFactor = Short.parseShort(config.get("default.replication.factor").value());
+
+                // Create topic
+                final NewTopic topic = new NewTopic(topicName, (short)1, replicationFactor);
+                topic.configs(Collect.hashMapOf("cleanup.policy", "delete", "retention.ms", Long.toString(MESSAGE_RETENTION_IN_DAYS.toMillis())));
+                admin.createTopics(Collections.singleton(topic));
+                logger.info("Database history topic created");
+            } else {
+                logger.info("Database history topic already exists");
+            }
+        } catch (Exception e) {
+            logger.warn("Debezium could not create database history topic, delegating to operator or broker", e);
```

Comment:
Just a minor nit, but I wouldn't mention "Debezium" in messages, as that's always the source of any log messages:

> Creation of database history topic failed, please create the topic manually